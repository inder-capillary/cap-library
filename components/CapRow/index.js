/**
* CapRow
*/
import React from "react";
import PropTypes from 'prop-types';
import { Row } from "antd";
import "./_capRow.scss";
const classNames = require('classnames');

class CapRow extends React.Component { // eslint-disable-line react/prefer-stateless-function
  render() {
    const {className, children, ...rest} = this.props;
    return (
      <Row {...rest} className={classNames("cap-row-v2", className)}>
        {React.Children.toArray(children)}
      </Row>
    );
  }
}

CapRow.propTypes = {
  children: PropTypes.any,
  className: PropTypes.string,
};

export default CapRow;
