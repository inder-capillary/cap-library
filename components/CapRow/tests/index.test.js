import renderer from 'react-test-renderer';
import React from 'react'; // eslint-disable-line no-unused-vars

import CapRow from '../index';

describe('<CapRow />', () => {
  it('CapRow renders correctly', () => {
    const tree = renderer.create(<CapRow />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
