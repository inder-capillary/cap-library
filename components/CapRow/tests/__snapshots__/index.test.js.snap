// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CapRow /> CapRow renders correctly 1`] = `
<div
  className="ant-row cap-row-v2"
/>
`;
